.github/workflows/tests.yml

```
@@ -0,0 +1,63 @@
+      matrix:
+        php: [7.4, 8.0, 8.1, 8.2]
+        laravel: [8.*, 9.*, 10.*]
+        exclude:
+          - php: 7.4
+            laravel: 10.*
+
+    name: P${{ matrix.php }} - L${{ matrix.laravel }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)